frontend/src/utils/getJwtEmail.ts

```diff
@@ -0,0 +1,23 @@
+import * as access from 'wildebeest/backend/src/access'
+
+export function getJwtEmail(jwtCookie: string) {
+	let payload: access.JWTPayload
+	if (!jwtCookie) {
+		throw new Error('Missing Authorization')
+	}
+	try {
+		// TODO: eventually, verify the JWT with Access, however this
+		// is not critical.
```

Comment:
@xtuc is this still not critical?

Reply:
Seems like it is now that we have login. I'd think you'd want a call to `checkAuth` or something similar here

Reply:
this PR is low risk I think as it doesn't allow access to the pages anyways
(although the endpoints are enable), so maybe it's ok to merge it in and let @xtuc look into this?